springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/serializers/KafkaChannelBindingSerializer.java

```diff
@@ -0,0 +1,27 @@
+package io.github.stavshamir.springwolf.asyncapi.serializers;
```

Comment:
I added it because the endpoint wouldn't work, it resulted in a 500 internal error failing to serialize the channelBinding field. Is there other way to fix this problem?

Reply:
Hmm ok, so let's keep it, I can figure it out later.